streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -789,6 +791,13 @@ private static ScheduledExecutorService maybeCreateRocksDBMetricsRecordingServic
         return null;
     }
 
+    private static void maybeWarnAboutCodeInRocksDBConfigSetter(final Logger log,
```

Comment:
I like the addition, we can reuse this when they inevitably do something else we'll need to warn our users about 😜